test/api/junction_detection_test.cpp

```diff
@@ -32,7 +32,6 @@ TEST(junction_detection, fasta_out_not_empty)
         "Reference\tchr21\t41972616\tForward\tRead\t0\t2294\tForward\t1\n"
         "Reference\tchr21\t41972616\tReverse\tRead\t0\t3975\tReverse\t1\n"
         "Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\t1\n"
-        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\t2\n"
```

Comment:
Do you understand why this line is removed from all of the test cases? I would not expect a change in output based on your changes.

Reply:
I don't know. I was confused aswell. But after my rebase, this wierd change disapeard again. Must be a mistake I made, which got removed now. Good that we have tests :)